src/plugins/notebook/utils/notebook-entries.js

```diff
@@ -8,6 +7,33 @@ const TIME_BOUNDS = {
     END_DELTA: 'tc.endDelta'
 };
 
+export function addEntryIntoPage(notebookStorage, entries, entry) {
+    if (!notebookStorage) {
+        return null;
+    }
+
+    const defaultSection = notebookStorage.section;
+    const defaultPage = notebookStorage.page;
+    if (!defaultSection || !defaultPage) {
+        return null;
```

Comment:
I think we should be consistent with the rest of our codebase and only use `undefined`.